client/verta/verta/_internal_utils/_pip_requirements_utils.py

```diff
@@ -267,34 +277,100 @@ def add_verta_and_cloudpickle(requirements):
         conflicts with the version in the current environment.
 
     """
-    # add verta
-    verta_req = "verta=={}".format(__about__.__version__)
-    for req in requirements:
-        if req.startswith("verta"):  # if present, check version
-            our_ver = verta_req.split('==')[-1]
-            their_ver = req.split('==')[-1]
-            if our_ver != their_ver:  # versions conflict, so raise exception
-                raise ValueError("Client is running with verta v{}, but the provided requirements specify v{};"
-                                 " these must match".format(our_ver, their_ver))
-            else:  # versions match, so proceed
-                break
-    else:  # if not present, add
-        requirements.append(verta_req)
-
-    # add cloudpickle
-    cloudpickle_req = "cloudpickle=={}".format(cloudpickle.__version__)
-    for req in requirements:
-        if req.startswith("cloudpickle"):  # if present, check version
-            our_ver = cloudpickle_req.split('==')[-1]
-            their_ver = req.split('==')[-1]
-            if our_ver != their_ver:  # versions conflict, so raise exception
-                raise ValueError("Client is running with cloudpickle v{}, but the provided requirements specify v{};"
-                                 " these must match".format(our_ver, their_ver))
-            else:  # versions match, so proceed
-                break
-    else:  # if not present, add
-        requirements.append(cloudpickle_req)
+    for i, req in enumerate(requirements):
+        if req.startswith("-e git+git@github.com:VertaAI/modeldb.git"):
```

Comment:
Do we need to check for the ssh style too?

Reply:
@conradoverta Hmm maybe?

My intention here was just to handle a particular case where `pip freeze` outputs something that `pip install` syntactically cannot handle. As far as I've seen, ssh style lines are syntactically valid, so I wouldn't think to check for it.

Is the concern around having the ssh keys in the model container?

Reply:
No, just wasn't exactly sure it supported ssh. LGTM then =)